bigtable/google/cloud/bigtable/row.py

```diff
@@ -236,6 +239,19 @@ def _delete_cells(self, column_family_id, columns, time_range=None,
             mutations_list.extend(to_append)
 
 
+def wrapped_partial(func, *args, **kwargs):
```

Comment:
Other APIs have been happy with just using `functools.partial` and not needing to also use `update_wrapper`. Can you justify why this is needed here over just using `partial`?